lib/ansiblelint/utils.py

```diff
@@ -25,7 +25,7 @@
 
 import ansible.constants as C
 from ansible.errors import AnsibleError
-from ansible.module_utils.splitter import split_args
+from ansible.parsing.splitter import split_args
```

Comment:
Does this work with ansible 1.9? Not sure I'm totally ready to abandon ansible 1.9 yet (unless Ansible themselves have)